app/src/main/java/fr/free/nrw/commons/quiz/QuizActivity.java

```diff
@@ -122,34 +111,36 @@ R.drawable.ic_error_outline_black_24dp, getTheme()))
     public void evaluateScore() {
         if ((quiz.get(questionIndex).isAnswer() && isPositiveAnswerChecked) ||
                 (!quiz.get(questionIndex).isAnswer() && isNegativeAnswerChecked) ){
-            customAlert(getResources().getString(R.string.correct),quiz.get(questionIndex).getAnswerMessage() );
+            customAlert(getResources().getString(R.string.correct),
+                quiz.get(questionIndex).getAnswerMessage());
             score++;
         } else {
-            customAlert(getResources().getString(R.string.wrong), quiz.get(questionIndex).getAnswerMessage());
+            customAlert(getResources().getString(R.string.wrong),
+                quiz.get(questionIndex).getAnswerMessage());
         }
     }
 
     /**
      * to display explanation after each answer, update questionIndex and move to next question
-     * @param title
-     * @param Message
+     * @param title the alert title
+     * @param Message the alert message
      */
-    public void customAlert(String title, String Message) {
-        AlertDialog.Builder alert = new AlertDialog.Builder(this);
-        alert.setTitle(title);
-        alert.setMessage(Message);
-        alert.setPositiveButton(R.string.continue_message, (dialog, which) -> {
-            questionIndex++;
-            if (questionIndex == quiz.size()) {
-                Intent i = new Intent(QuizActivity.this, QuizResultActivity.class);
-                dialog.dismiss();
-                i.putExtra("QuizResult",score);
-                startActivity(i);
-            } else {
-                displayQuestion();
-            }
-        });
-        AlertDialog dialog = alert.create();
-        dialog.show();
+    public void customAlert(final String title, final String Message) {
+        new AlertDialog.Builder(this)
+            .setTitle(title)
+            .setMessage(Message)
+            .setPositiveButton(R.string.continue_message, (dialog, which) -> {
+                questionIndex++;
+                if (questionIndex == quiz.size()) {
+                    final Intent intent = new Intent(this, QuizResultActivity.class);
+                    dialog.dismiss();
+                    intent.putExtra("QuizResult", score);
+                    startActivity(intent);
+                } else {
+                    displayQuestion();
+                }
+            })
+            .create()
+            .show();
```

Comment:
Similar to the other alert dialog - followed the builder pattern and inlined variables to reduce duplication.